### PR TITLE
Pol 316 add savings aws old snapshots

### PR DESCRIPTION
**Pull request by gopinathmahakud1**
### Description

Added savings in the incidents

### Issues Resolved

- Include Estimated Monthly Savings to each resource
- Include Total Estimated Monthly Savings in the incident summary

### Contribution Check List

- [x] All tests pass.
- [x] New functionality includes testing.
- [x] New functionality has been documented in the README if applicable
- [x] New functionality has been documented in CHANGELOG.MD

https://governance.rightscale.com/org/1105/projects/60073/policy-incidents/5f452e60256320000191fe4a
image

### CODE REVIEW

**Comment by gopinathmahakud1** [comment]
I have fixed the conflict and committed the changes. Thanks

File: cost/aws/old_snapshots/README.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 ## What it does
 
-This policy finds AWS snapshots in the given account which are older than the specified days and deletes them after user approval. For a snapshot, if images are created then we can't delete the snapshot without deleting the images. So if the user selects Yes, the snapshot will be deleted along with the images, and if No the snapshot will not be considered for deletion. Account specific snapshots are determined by filtering based on the owner-id. The account number is used as an owner-id.
+This policy finds AWS snapshots in the given account which are older than the specified days and deletes them after user approval. For a snapshot, if images are created then we can't delete the snapshot without deleting the images. So, if the user selects Yes, the snapshot will be deleted along with the images, and if No the snapshot will not be considered for deletion. Account specific snapshots are determined by filtering based on the owner-id. The account number is used as an owner-id.
```
Comment (cdwilhelm) [inline cost/aws/old_snapshots/README.md:5]: please add Estimated Savings detail to the readme.  See the POL epic for an example.  I'll update the other policies with the new readme content.

File: cost/aws/old_snapshots/README.md [hunk 2]
```diff
@@ -2,7 +2,13 @@
...
+
+### Policy savings details
+
+The policy includes the estimated savings. The estimated savings is recognized if the resource is terminated. Optima is used to receive the estimated savings which is the product of the most recent full day’s cost of the resource * 30. The savings is displayed in the Estimated Monthly Savings column. If the resource can not be found in Optima the value is n/a. The incident header includes the sum of each resource Estimated Monthly Savings in the Incident Header as Total Estimated Monthly Savings.
+
+The Estimated Savings and Estimated Monthly Savings rounded to 3 decimal places, so the savings value will display $0.000 if estimated savings is less than $0.0005.
```
Comment (cdwilhelm) [inline cost/aws/old_snapshots/README.md:11]: should this be Estimated Monthly Saving and Total Estimated Monthly savings?

**Review by cdwilhelm — CHANGES_REQUESTED** [review]

**Review by cdwilhelm — DISMISSED** [review]
LGTM, thanks

**Review by cdwilhelm — APPROVED** [review]

File: cost/aws/old_snapshots/CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v2.6
```
Comment (douglaswth) [inline cost/aws/old_snapshots/CHANGELOG.md:3]: This version bump was not actually included in the actual Policy Template file, so it probably wasn't published.

**Pull request merged**